Documentation/project-docs/pullrequest-builds.md

```diff
@@ -0,0 +1,92 @@
+# Pull Request Builds
+
+As part of our Pull Requests we have some validation builds where we build the product and run tests on different Operative Systems and Build Configurations.
+
+### Build Matrix
+
+| Build | Framework | Builds Tests | Runs Tests |
+|:---|:---:|:---:|:---:|
+| Windows x86_Release | netcoreapp | X | X |
+| Windows x64_Debug | netcoreapp | X | X |
+| Windows NETFX_x86_Release | netfx | X | X |
+| Windows UWP_CoreCLR_x64_Debug | uap | X | X |
+| Windows UWP_NETNative_x86_Release | uapaot | X |   |
+| Linux x64_Release | netcoreapp | X | X |
+| Linux arm64_Release | netcoreapp | X | X |
+| Linux arm_Release | netcoreapp | X |   |
+| Linux musl_x64_Debug | netcoreapp | X |   |
+| MacOS x64_Debug | netcoreapp | X |   |
+| Packaging All Configurations x64_Debug | all | X | X |
+
+Our build definitions are defined by some .yml files with the following structure:
+
+- Entrypoint `.azure-pipelines.yml`
+- Shared build steps `eng/pipelines/corefx-base.yml`
+- Helix step `eng/pipelines/helix.yml`
+- Windows Matrix `eng/pipelines/windows.yml`
+- Linux Matrix `eng/pipelines/linux.yml`
+- MacOS Matrix `eng/pipelines/macos.yml`
+
+## Available builds
+
+- corefx-ci (Runs by default on every commit)
+- corefx-outerloop-linux
+- corefx-outerloop-osx
+- corefx-outerloop-windows
+
+### Azure pipelines commands
+
+```
+/azp help (displays help message)
+/azp run (runs all builds listed above)
+/azp run <build name> (/azp run corefx-outerloop-linux)
+```
+
+`/azp` is a shortcut for `/AzurePipelines` which can be used as well.
+
+In order to use comment triggers, at the moment you need to be a `Contributor` or `Member` of the repo.
```

Comment:
Should this be "Collaborator" over "Contributer"? I seem to recall you somewhere saying AZP comments only work for collaborators (maybe I am misremembering...)

Reply:
Yeah you're correct! Thanks for remembering.